packages/js/ai/src/hooks/useBackgroundRemoval.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import apiFetch from '@wordpress/api-fetch';
+
+export type BackgroundRemovalParams = {
+	imageFile: File;
+	returnPngImage?: boolean;
```

Comment:
This locks us into file formats of just a png, or the default. I'm not sure what else we might support, but maybe we should just have a `format` prop with possible value `png | jpg` and a default of `jpg`?

Reply:
Thanks, this is exactly what we should be doing to have a less tightly coupled integration. I've addressed this and other feedback in https://github.com/woocommerce/woocommerce/pull/40089/commits/1b8f146a80b89970cca4e90e44f914daa1423c8a